kebechet/managers/version/version.py

```diff
@@ -226,8 +226,29 @@ def run(  # type: ignore
                         ["git", "push", "origin", f"refs/tags/{tag_version}"]
                     )
                 )
-            return
+                try:
+                    # Branch name is same as tag version
+                    branch_to_delete = tag_version
```

Comment:
can directly use the tag_version ? instead of creating another variable?

Reply:
ACK!